jwala-integration-test/src/test/java/com/cerner/jwala/ui/selenium/SeleniumTestHelper.java

```diff
@@ -131,4 +121,78 @@ public static void runSqlScript(final String sqlScript) throws IOException, Clas
             }
         }
     }
+
+    public static void checkServiceDeleteWasSuccessful(String serviceName, JschService jschSvc, Properties properties) {
+        LOGGER.info("checkServiceDeleteWasSuccessful {}", serviceName);
+
+        props = properties;
+        jschService = jschSvc;
+
+        // indirectly required by JschServiceImpl via use of ApplicationProperties
+        final String originalPropertiesRootPath = System.getProperty("PROPERTIES_ROOT_PATH");
+        System.setProperty("PROPERTIES_ROOT_PATH", SeleniumTestHelper.class.getResource("/selenium/vars.properties").getPath()
+                .replace("/vars.properties", ""));
+
+        final String hostname = props.getProperty("host1");
+
+        final RemoteSystemConnection remoteSystemConnection = getRemoteSystemConnection(hostname);
+
+        final JwalaOsType osType = getJwalaOsType(remoteSystemConnection);
+
+        if (osType.equals(JwalaOsType.WINDOWS)) {
+            checkWindowsService(serviceName, hostname, remoteSystemConnection);
+        } else {
+            checkLinuxServiceRunLevel(serviceName, hostname, remoteSystemConnection);
+            checkLinuxService(serviceName, hostname, remoteSystemConnection);
+        }
+
+        // Restore the properties root path
+        if (null != originalPropertiesRootPath) {
+            System.setProperty("PROPERTIES_ROOT_PATH", originalPropertiesRootPath);
+        }
+    }
+
+    private static void checkLinuxService(String serviceName, String hostname, RemoteSystemConnection remoteSystemConnection) {
+        RemoteCommandReturnInfo remoteCommandReturnInfo = jschService.runShellCommand(remoteSystemConnection, "sudo service " + serviceName + " status", SHORT_CONNECTION_TIMEOUT);
```

Comment:
Can the lines be shortened ? Not 80 but enough for it to fit github review screen perhaps ?